### ISSUE TITLE
Event Namespacing

### ISSUE THREAD

**Issue by shshaw** [kenwheeler/cash#121]
As mentioned in [#114](https://github.com/kenwheeler/cash/issues/114#issuecomment-217455552), event namespacing would be a worthwhile addition. My changes in the [event-callbacks](https://github.com/kenwheeler/cash/tree/event-callbacks) branch should make this more easily doable.

```
$('body').on('click', fn);
$('body').on('keydown.myNamespace', fn);
$('body').on('scroll.myNamespace', fn);
$('body).off('.myNamespace');
```

**Comment by shshaw** [comment]
Looks like jQuery's namespacing supports multiple namespaces, for example: 

```js
$el.on('click.myNamespace.specificNamespace',fn);
$el.off('.specificNamespace'); // Removes fn callback from click.
```

Is this something we need to support? @kenwheeler @simeydotme

**Comment by tjbenton** [comment]
If you're still trying to keep cash compatible with Bootstrap it should support multiple name spaces, because Bootstrap uses multiple namespaces all over the place.

**Comment by shshaw** [comment]
Good to know, @tjbenton. I'll keep that in mind when I get some time to return to the Event setup.

**Comment by fabiospampinato** [comment]
This is the single biggest thing preventing me from switching from jQuery. Is there an approximate ETA?

**Comment by stvhanna** [comment]
Any updates on this?

**Comment by axten** [comment]
we also strongly need this feature to switch to cash in our projects.

**Comment by fabiospampinato** [comment]
For anyone interested in this, I've implement this in my [fork](https://github.com/fabiospampinato/cash), along with some other enhancements related to events (data passed as argument, ability to remove all events from an element, proper support for removing wrapped callbacks).

**Comment by fabiospampinato** [comment]
Fixed in v2.1.2.
